test-lib.sh

```diff
@@ -816,6 +816,8 @@ ct_get_public_image_name() {
     public_image_name=$registry/centos7/$base_image_name-${version//./}-centos7:centos7
```

Comment:
Same here

Reply:
This will be part of different PR, that does not exist yet ;) RHEL-7 and CentOS 7 deprecation.